src/app/newSettings/Sections/Account/Plans/PlansSection.tsx

```diff
@@ -39,17 +39,25 @@ const PlansSection = ({ changeSection }: PlansSectionProps) => {
 
   const plan = useSelector<RootState, PlanState>((state) => state.plan);
   const user = useSelector<RootState, UserSettings | undefined>((state) => state.user.user);
-  const { subscription: currentUserSubscription } = plan;
-  const currentPlan = plan.individualPlan;
-  const currentPlanPayPeriod = currentPlan?.renewalPeriod === 'monthly' ? 'month' : 'year';
+  const { subscriptionIndividual, subscriptionBusiness } = plan;
   let stripe;
 
   if (user === undefined) throw new Error('User is not defined');
 
   const [isLoadingCheckout, setIsLoadingCheckout] = useState(false);
-  const [prices, setPrices] = useState<DisplayPrice[]>([]);
+
+  const [individualPrices, setIndividualPrices] = useState<DisplayPrice[]>([]);
+  const [businessPrices, setBusinessPrices] = useState<DisplayPrice[]>([]);
+
+  const [selectedIntervalSubscription, setSelectedIntervalSubscription] = useState<'individual' | 'business'>(
```

Comment:
rather than interval it is the type of subscription, I will use a more related name